### DISCUSSION TITLE
How to create framework in order to achieve this hierarchy?

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Knowlg/Community#21] [Q&A] [answered]
Expected: 
![image](https://user-images.githubusercontent.com/22250006/173815406-d7769c3f-0f9f-4a24-a4fc-39501b4c7001.png)

Existing:
![image](https://user-images.githubusercontent.com/22250006/173815463-bb9515a9-0cdf-4268-a44e-49d73ec5821f.png)

We have created the framework as follows:
https://pastebin.com/R3tUQbt0

Expected Values in the dropdown
Category:
Civil, Health, Accounts

Sub-Category:
Knowledge, Mission, Software

Language:
English, Hindi, Marathi

Category and Sub-category are not dynamic-dependent dropdowns.

The above-created framework is not working properly. What changes are to be done in the framework hierarchy to achieve the effect?

@vinukumar-vs 

cc: @komalm

**Comment by pranavirabatti** [comment, marked as answer]
Any update on this?

**Reply by Krishnaj20** [reply]
@pranavirabatti - You will have to reach out to SunbirdEd team - this looks like an issue with the Sunbird consumption web app. 

@luckynvdu @sharathkashyap
